### PR TITLE
perf(kiali): add context, remove kiali-common and refactor backend

### PR DESCRIPTION
**Pull request by aljesusg**
# Kiali Major version

- [X] Remove kiali-backend plugin
- [X] Remove kiali-common plugin
- [X] Move types to kiali
- [X] Move logic to kiali plugin
- [X] Change API to proxy configuration
- [X] Add alert system
- [X] Add state with context
- [X] Improve dev environment
- [x] Add docs about configuration
- [x] Fix lint
- [x] Fix tsc

### CODE REVIEW

**Comment by invincibleJai** [comment]
Thanks @aljesusg, good to add screenshots/gif's/video as applicable on the description and good to add Owners file for each of the plugin i.e https://github.com/janus-idp/backstage-plugins/blob/main/plugins/feedback/OWNERS

**Comment by aljesusg** [comment]
> Thanks @aljesusg, good to add screenshots/gif's/video as applicable on the description and good to add Owners file for each of the plugin i.e https://github.com/janus-idp/backstage-plugins/blob/main/plugins/feedback/OWNERS 

Thanks!

There is a screenshot added in the documentation. Rebased for OWNERS file

File: plugins/kiali/package.json [hunk 1]
```diff
@@ -31,13 +31,26 @@
...
-    "react-use": "^17.4.0"
+    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-icons": "^5.1.1",
+    "axios": "^1.5.1",
+    "cytoscape": "3.15.5",
+    "deep-freeze": "0.0.1",
+    "history": "^5.3.0",
+    "json-beautify": "1.0.1",
+    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
```
Comment (invincibleJai) [inline plugins/kiali/package.json:47]: do we need both?

Comment (aljesusg) [reply plugins/kiali/package.json:47]: weird, maybe yarn installed that by the prompt let me remove lodash, we used lodash-es

Comment (aljesusg) [reply plugins/kiali/package.json]: Well we duplciate the libs in kiali and it seems that we pulled lodash-es worngly from our standalone let me fix it

**Review by invincibleJai — APPROVED** [review]
/lgtm

**Pull request merged**